src/core/embedders/StandardFontEmbedder.ts

```diff
@@ -81,6 +82,21 @@ class StandardFontEmbedder {
     return (1000 * height) / (yTop - yBottom);
   }
 
+  getFontMetrics() {
+    const bbox = this.font.FontBBox;
+    return {
+      ascent: this.font.Ascender,
+      descent: this.font.Descender,
+      // It would be nice to figure out how to get this value
+      lineGap: 0,
```

Comment:
We should set this value to `undefined` rather than `0` so users can tell when it is actually `0` vs simply not being available.

Reply:
My reason for setting it to 0 is that a line gap of 0 will tend to be reasonable for nearly all fonts -- in fact, it's very likely that it's reasonable for all of the standard fonts. It is in fact the line gap which pdf-lib is using as far as I can tell, since I don't believe you are adding any arbitrary line gap.

Reply:
That seems reasonable to me 👍